bin/commands/version/.help

```diff
@@ -1 +1,6 @@
-Display Zowe version.
+Synopsis
+    zwe version [parameter]...
```

Comment:
is there a parameter? I don't see one

Reply:
version doesn't have its own parameters but it can be executed with global parameters like 
zwe version -h
zwe version -v